frontend/src/Login/authReducer.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN } from './actionCreator';
+
+const defaultState = {
+  isLogin: false,
+};
+
+export default function auth(state = defaultState, action) {
+  switch (action.type) {
+    case LOGIN:
+      return { ...state, isLogin: true };
```

Comment:
```suggestion
      return { ...state, isLoggedIn: true };
```
I guess you wanted this, right?